src/product-card.css

```diff
@@ -0,0 +1,94 @@
+.container {
+  width: 198px;
+  box-sizing: border-box;
+  padding: 0 16px;
+}
+
+.container--card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.container__picture {
+  font-family: "Roboto", sans-serif;
+  width: 160px;
+  margin-left: 3px;
+  margin-right: 19px;
+  margin-top: 32px;
+  margin-bottom: 40px;
```

Comment:
try to set height and width for picture, instead of using all these custom margins